src/nuspacesim/simulation/taus/local_plots.py

```diff
@@ -123,3 +143,104 @@ def taus_pexit(inputs, results, *args, **kwargs):
+    )
+    binning_e = np.arange(
+        np.round(np.min(np.log10(showerEnergy)), 1) - 0.1,
+        np.round(np.max(log_e_nu), 1) + 0.1,
+        0.1,
+    )
+    ax[0, 0].hist(
+        x=log_e_nu, bins=binning_e, color="C0", alpha=0.6, label=r"$E_{\nu_\tau}$"
+    )
+    ax[0, 0].hist(
+        x=np.log10(tauEnergy),
+        bins=binning_e,
+        color="C1",
+        alpha=0.6,
+        label=r"$E_\tau$",
+    )
+    ax[0, 0].hist(
+        x=np.log10(showerEnergy),
+        bins=binning_e,
+        color="C2",
+        alpha=0.6,
+        label=r"$E_\mathrm{shower}$",
+    )
+    ax[0, 0].set_yscale("log")
+    ax[0, 0].legend(loc="upper left")
+    ax[0, 0].set_xlabel(
+        r"Energy / $\log_\mathrm{10}\left(\frac{E}{\mathrm{eV}}\right)$"
+    )
+    ax[0, 0].set_ylabel(r"Counts")
+
+    n, bins, _ = ax[0, 1].hist(x=np.degrees(betas), bins=binning_b)
+    ax[0, 1].set_xlabel(r"Earth emergence angle $\beta$ / $^{\circ}$")
+    ax[0, 1].set_ylabel("Counts")
+    ax[0, 1].set_yscale("log")
+    ax[0, 1].set_xlim(min(binning_b), max(binning_b))
+
+    binning_y = np.logspace(
+        np.log10(np.min(tauLorentz)) - 0.2, np.log10(np.max(tauLorentz)) + 0.2, 25
+    )
+    _, _, _, im = ax[1, 0].hist2d(
+        np.degrees(betas), tauLorentz, bins=[binning_b, binning_y], cmin=1
+    )
+    bincenter, mean, std, binwidth = get_profile(
+        np.degrees(betas), tauLorentz, 10, useStd=True, **kwargs
+    )
+    ax[1, 0].errorbar(
+        bincenter,
+        mean,
+        yerr=std,
+        xerr=binwidth,
+        color="C1",
+        fmt=".",
+        lw=2,
+        zorder=5,
+        label="Profile",
+    )
+    divider = make_axes_locatable(ax[1, 0])
+    cax = divider.append_axes("right", size="5%", pad=0.0)
+    cbar = fig.colorbar(im, cax=cax)
+    cbar.set_label("Counts")
+    ax[1, 0].set_yscale("log")
+    ax[1, 0].set_xlim(min(binning_b), max(binning_b))
+    ax[1, 0].legend(loc="upper center")
+    ax[1, 0].set_xlabel(r"Earth emergence angle $\beta$ / $^{\circ}$")
+    ax[1, 0].set_ylabel(r"$\tau_\mathrm{Lorentz}$")
+
+    binning_y = np.logspace(
+        np.log10(np.min(tauExitProb)) - 0.2, np.log10(np.max(tauExitProb)) + 0.2, 25
+    )
+    _, _, _, im = ax[1, 1].hist2d(
+        np.degrees(betas), tauExitProb, bins=[binning_b, binning_y], cmin=1
+    )
+    divider = make_axes_locatable(ax[1, 1])
+    cax = divider.append_axes("right", size="5%", pad=0.0)
+    cbar = fig.colorbar(im, cax=cax)
+    cbar.set_label("Counts")
+    ax[1, 1].set_yscale("log")
+    ax[1, 1].set_xlim(min(binning_b), max(binning_b))
+    ax[1, 1].set_xlabel(r"Earth emergence angle $\beta$ / $^{\circ}$")
+    ax[1, 1].set_ylabel(r"$P_\mathrm{Exit}(\tau)$")
+
+    fig.suptitle("Overview of Tau interaction properties")
+    fig.tight_layout()
+    plt.show()
+    if "pop_up" in kwargs and kwargs.get("pop_up") is True:
+        plt.show()
+    fig.savefig("taus_overview", format="pdf")
```

Comment:
Should be able to turn off savefig in the config.ini file.
Also needs to read the extension from the config file rather than default to 'pdf'

Reply:
Should be able to disable saving images in the config file. Also consider pulling that functionality out into a single function, so future plot functions don't need to re-implement it inline. Keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Reply:
Agreed! I am going to work on implementing these features in the coming weeks. But since we wanted to have the plot-config feature available for the 1.0 release I could just remove this part 
`if "pop_up" in kwargs and kwargs.get("pop_up") is True:
        plt.show()
    fig.savefig("taus_overview", format="pdf")`
of the code for now?
It's probably also better to remove the not-yet implemented functionalities in 'sample_plot_config.ini'

Reply:
Yeah that's a good overview of the priorities.